.github/workflows/CI.yml

```diff
@@ -98,107 +95,23 @@ jobs:
         name: wheels
         path: dist
 
-  release:
-    name: Release
+  upload:
+    name: Store wheels in S3
     runs-on: ubuntu-latest
-    if: "startsWith(github.ref, 'refs/tags/')"
-    needs: [ macos, windows, linux ]
-    outputs:
-      bytewax-version: ${{ steps.get-info.outputs.bytewax-version }}
-      version-image-tag: ${{ steps.get-info.outputs.version-image-tag }}
-      latest-image-tag: ${{ steps.get-info.outputs.latest-image-tag }}
+    if: "github.ref == 'refs/heads/main'"
+    needs: [ linux ]
```

Comment:
Does this need all the different platforms to have finished building their wheels?

Reply:
yeah, I missed the other platforms. Now is fixed, thank you.